### DISCUSSION TITLE
Would be a good approach to constraining a control to the “bottom” of a floating panel regardless of its height?

### DISCUSSION THREAD

**Discussion by AustinConlon** [scenee/FloatingPanel#484] [Q&A] [answered]
If I were to constrain a toolbar button to be just above the bottom safe area, in a floating panel that I only use in a half-screen format, then the floating panel view controller is still treated like a full screen modal. This means scrolling is required to find the button. How would I instead anchor it to the bottom and instead use strictly a half-screen sheet?

**Comment by scenee** [comment, marked as answer]
[`fitToBounds` mode](https://github.com/scenee/FloatingPanel#scale-the-content-view-when-the-surface-position-changes) probably meets your use case.
